components/service/fretboard/src/main/java/mozilla/components/service/fretboard/ValuesProvider.kt

```diff
@@ -0,0 +1,87 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.fretboard
+
+import android.content.Context
+import android.os.Build
+import java.util.Locale
+
+/**
+ * Class used to provide
+ * custom filter values
+ */
+open class ValuesProvider {
+    /**
+     * Provides the user's language
+     *
+     * @return language
```

Comment:
Let's either add an explanation here or remove the "@return language" doc because it doesn't add anything.